notification-lambda-cfn.yaml

```diff
@@ -90,4 +93,22 @@ Resources:
       FunctionName: !GetAtt Function.Arn
       Action: lambda:InvokeFunction
       Principal: events.amazonaws.com
-      SourceArn: !GetAtt EventRule.Arn
\ No newline at end of file
+      SourceArn: !GetAtt EventRule.Arn
+
+  LambdaAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmActions: [!Ref AlarmTopic]
+      OKActions: [!Ref AlarmTopic]
+      AlarmDescription: !Sub Triggers if there are errors persisiting on ${Stage}
+      ComparisonOperator: GreaterThanThreshold
+      EvaluationPeriods: 1
+      MetricName: "Errors"
+      Namespace: "AWS/Lambda"
+      Dimensions:
+        - Name: "FunctionName"
+          Value: !Sub us-election-2020-notification-lambda-${Stage}
+      Period: 3600
```

Comment:
This evaluation period seems quite long - perhaps we can shorten it so that we can react a little more quickly?